src/scss/grommet-core/_objects.tabs.scss

```diff
@@ -5,16 +5,33 @@
   padding: 0;
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
   align-items: center;
   list-style: none;
   border-bottom: 1px solid $border-color;
 
   &__content {
-    padding: $inuit-base-spacing-unit;
-
     @include media-query(palm) {
       padding: $inuit-base-spacing-unit 0;
     }
   }
+
+  &--justify-center {
+    justify-content: center;
+  }
+
+  &--justify-start {
+    justify-content: flex-start;
+
+    @include media-query(portable) {
```

Comment:
Still left-justified for small screens:
<img width="464" alt="screen shot 2016-02-10 at 9 17 52 am" src="https://cloud.githubusercontent.com/assets/10161095/12958749/517854f8-cfd7-11e5-9310-aaee8e2bdcfc.png">
Center-justified for small screens:
<img width="446" alt="screen shot 2016-02-10 at 9 18 12 am" src="https://cloud.githubusercontent.com/assets/10161095/12958755/577769a2-cfd7-11e5-8e36-e8ae35d15a53.png">

I think the latter looks better if the screen is too small to support all tabs on a single row. What do you think?